src/redux-query-sync.js

```diff
@@ -99,6 +99,7 @@ function ReduxQuerySync({
                 locationParams.delete(param)
             } else {
                 locationParams.set(param, valueToString(value))
+                lastQueryValues[param] = value
             }
```

Comment:
If I am not mistaken, you could add this line right here below the if/else statement. Also if the parameter is deleted, we should remember the value, no?